scripts/cleanup-test-processes.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# This script can be handy to kill test processes launched manually
+# for debugging or still hanging around for other reasons.
+
+pkill --uid "${UID}" --full '/monerod .* --regtest '
+pkill --uid "${UID}" --full '/ganache.* --deterministic '
+
+# If you have monero-wallet-rpc or swapd processes owned by the current user
+# that you don't want to kill, don't use this script!
+pkill --uid "${UID}" --full '/monero-wallet-rpc '
+pkill --uid "${UID}" --full '/swapd '
```

Comment:
This script is helpful for me at least. If you don't think it is useful, I can get rid of it and just use a bash function from my startup scripts.

Reply:
this is helpful!